src/checks/kubernetes/services/ports/ports.go

```diff
@@ -38,12 +39,30 @@ type doesPortExistStruct struct {
 	} `yaml:"values"`
 }
 
+func doesPortExistParse(valuesYaml string, v *doesPortExistStruct) error {
+
+	err := yaml.Unmarshal([]byte(valuesYaml), &v)
+	if err != nil {
+		log.Fatalf("Unmarshal: %v", err)
```

Comment:
Just a nitpick, I would pass the error down since you have a `log.FatalF` that would pick it up.

Reply:
Yeah...you are correct...i should do that.  Also...one of the tickets says that these types of errors should just be in the output table on why it failed.  Ill fix this.

Reply:
I have changed it so that the error thrown goes into the check output.  For example...if someone didnt put in the `port` input values param and then ran the check:

```
+----------------------+----------------------------------------------------------------------+------+-------------+
| Test Type          | Name                                                                 | Pass | Message     |
+----------------------+----------------------------------------------------------------------+------+-------------+
| doesServicePortExist | Does microservice 1 have a kubernetes service with port 5000 exposed | false | Check values: no `Port` set  |
+----------------------+----------------------------------------------------------------------+------+-------------+
```